cartridge/health.lua

```diff
@@ -1,16 +1,28 @@
-local membership = require('membership')
-local argparse = require('cartridge.argparse')
+local membership = require("membership")
+local argparse = require("cartridge.argparse")
```

Comment:
Please don't change lines that don't connect to your pr directly

Reply:
Sure.